apps/server/src/modules/learnroom/common-cartridge/common-cartridge-web-link-resource.ts

```diff
@@ -20,23 +20,41 @@ export class CommonCartridgeWebLinkResourceElement implements ICommonCartridgeEl
 	}
 
 	content(): string {
-		return this.xmlBuilder.buildObject({
-			webLink: {
+		const extraTags = {
```

Comment:
Minor suggestion: I would call them commonTags, because they are used by both version. They are not additional tags.